src/main/resources/view/PinnedPersonCard.fxml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.FlowPane?>
+
+<VBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" alignment="CENTER" styleClass="pinned-person-item">
+    <HBox alignment="CENTER">
+        <Label fx:id="id" styleClass="cell_small_label">
+            <minWidth>
+                <!-- Ensures that the label text is never truncated -->
+                <Region fx:constant="USE_PREF_SIZE" />
+            </minWidth>
+        </Label>
+        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
```

Comment:
```suggestion
        <Label fx:id="name" text="\$firstName" styleClass="cell_big_label" />
```
Maybe this can be `firstName` for clarity?

Reply:
i think leaving it as name is better? Since we use the full name of the person rather than just their first name

Reply:
Oh ok then maybe use `\$name` instead? Note that I'm referring to the `text` attribute not the `fx:id`.

Reply:
Oh whoops sorry. I have just updated it in my new commit.